README.md

```diff
@@ -249,6 +249,46 @@ Default:        nil
 
 The `queryTimeout` parameter sets a timeout for the query. If the query takes longer than the timeout, it will be cancelled. If it is not set the default context timeout will be used.
 
+
+#### `roles`
+
+```
+Type:           string
+Format:         roles=catalog1=ROLE{role1},catalog2=ROLE{role2}
```

Comment:
Let's not expose the internal syntax to users. This was done in the Python driver, but now is considered a deprecated format. The JDBC driver should be our reference implementation, and it uses the `roles=catalog1:role1;catalog2:role2` syntax. This should match the syntax used for session properties, so you can reuse all the constants for separators, etc.